apis/controlplane/v1alpha4/nestedetcd_types.go

```diff
@@ -21,19 +21,23 @@ import (
 	addonv1alpha1 "sigs.k8s.io/kubebuilder-declarative-pattern/pkg/patterns/addon/pkg/apis/v1alpha1"
 )
 
+type NestedEtcdPhase string
+
+const (
+	NestedEtcdReady   NestedEtcdPhase = "ready"
+	NestedEtcdUnready NestedEtcdPhase = "unready"
+)
+
 // NestedEtcdSpec defines the desired state of NestedEtcd
 type NestedEtcdSpec struct {
-	// NestedComponentSpec contains the common and user-specified information that are
-	// required for creating the component
+	// NestedComponentSpec contains the common and user-specified information
+	// that are required for creating the component
 	// +optional
 	NestedComponentSpec `json:",inline"`
 }
 
 // NestedEtcdStatus defines the observed state of NestedEtcd
 type NestedEtcdStatus struct {
-	// Ready is set if all resources have been created
-	Ready bool `json:"ready,omitempty"`
```

Comment:
Do we not still want this?

Reply:
I think I see this was removed cause of the overlap with https://pkg.go.dev/sigs.k8s.io/kubebuilder-declarative-pattern/pkg/patterns/addon/pkg/apis/v1alpha1#CommonStatus

Reply:
No. I think we can use the `CommonStatus.Phase` instead.